### ISSUE TITLE
NTR: Liver biopsy: lipid-laden macrophages (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9213]
New term request Liver biopsy: lipid-laden macrophages (elthat_101212171010): Foamy cells on hepatic biopsy

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000159
- label: Hepatic lipid-laden macrophage infiltration
- parents: [Abnormal liver parenchyma morphology (HP:0030146)](https://hpo.jax.org/app/browse/term/HP:0030146)
- pmid: PMID:17939344
- def: Accumulation of foamy cells (lipid-laden macrophages) in the parenchyma of the liver.
- comment: See figure 2 of PMID:17939344.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9213
- synonyms: Foamy cells on hepatic biopsy(NONE)|Liver biopsy: lipid-laden macrophages(NONE)
